### ISSUE TITLE
Tabs & W3C standards

### ISSUE THREAD

**Issue by Eevolee** [Jahia/moonstone#1068]
### Keyboard interactions:
- Add left & right arrow navigation
- Home & End (optional)
### Attribute changes:
- Container div should have role="tablist"
- Each button should have role="tab"
- Each button should have aria-selected attribute

**Comment by ffffffelix** [comment]
- Create a new hook to handle navigation with the arrow Left and Right
- Do not apply Home and End
- Add all required attributes
